src/main/java/fi/metatavu/metamind/kuntaapi/KuntaApiConsts.java

```diff
@@ -0,0 +1,12 @@
+package fi.metatavu.metamind.kuntaapi;
+
+public class KuntaApiConsts {
+  
+  public static final String APIURL_SETTING_KEY = "kuntaapi-url";
+  public static final String USERNAME = "kuntaapi-username";
```

Comment:
Could be called for example APIUSER_SETTING_KEY and APIPASS_SETTING_KEY to keep sonar happy